packages/libs/react-ui/src/components/Link/Link.tsx

```diff
@@ -12,17 +13,23 @@ export interface ILinkProps {
   icon?: keyof typeof SystemIcon;
   iconAlign?: 'left' | 'right';
   asChild?: boolean;
+  inline?: boolean;
 }
 
 export const Link: FC<ILinkProps> = ({
   children,
   icon,
   iconAlign = 'left',
   asChild = false,
+  inline = false,
   ...restProps
 }) => {
   const Icon = icon && SystemIcon[icon];
 
+  const linkClasses = classnames(linkContainerClass, {
+    [inlineLinkClass]: inline,
+  });
+
```

Comment:
Should the link element be inline by default?

Reply:
ideally yes! but i made it as a separate prop so we could discuss and align and make changes